docs/image-sizes.md

```diff
@@ -12,6 +12,28 @@ They have default dimensions, but you can override these either with code or in
 
 ![Media Settings](./assets/media-settings.png)
 
+### Content Width
+
+If your theme defines a content width via the `$content_width` (or `$GLOBALS['content_width]`) global, this size will _override_ the size defined on the Settings > Media page. This will result in the Large size appearing to be the "wrong" size (using the size defined in the _global_ rather than the size in the admin) in the media modal. This global is intended to be used as an upper limit on the width of images based on your theme's design.
+
+For example, if your theme had a content area that was 1000 pixels wide, you might set the `$content_width` global in your theme like this:
+
+```php
+add_action( 'after_setup_theme', function () {
+	global $content_width;
+
+	if ( ! isset( $content_width ) ) {
+		$content_width = 1000;
+	}
```

Comment:
I don't think the `isset()` check is entirely necessary for this example. You'd generally want to intentionally override anyway I guess if it was already set.
```suggestion
	$content_width = 1000;
```

Reply:
The `isset` is what the default themes use so I was going off of that example.

Reply:
Ah ok. Soz, maybe it's a child theme thing 🤔  I'll try and remember not to jump the gun in future 🤠

Reply:
Yeah, child themes run their `functions.php` before the parent theme does, so that's probably why it's written that way. Should I open a PR to put it back, do you think?